src/Microsoft.VisualStudio.ProjectSystem.Managed/Resources.resx

```diff
@@ -181,9 +181,9 @@ If this file or folder does not appear in Solution Explorer, then it is not curr
   <data name="ProjectLoadedWithWrongProjectType" xml:space="preserve">
     <value>The project file '{0}' cannot be opened. 
 
-This project was loaded using the wrong project type, likely as a result of renaming the project extension outside of Visual Studio. Remove and readd the project to have the solution pick up the right project type.</value>
+This project was loaded using the wrong project type, likely as a result of renaming the project extension outside of Visual Studio. Remove and read the project to have the solution pick up the right project type.</value>
```

Comment:
This is talking about removing and then adding back the project, not reading the project :P

```suggestion
This project was loaded using the wrong project type, likely as a result of renaming the project extension outside of Visual Studio. Remove and re-add the project to have the solution pick up the right project type.</value>
```